deployments/data-hub/data-hub--staging.yaml

```diff
@@ -0,0 +1,230 @@
+          value: "False"
+        - name: INITIAL_S3_FILE_LAST_MODIFIED_DATE
+          value: "2020-02-14 13:00:00"
+        - name: MATERIALIZE_BIGQUERY_VIEWS_GCP_PROJECT
+          value: "elife-data-pipeline"
+        - name: MATERIALIZE_BIGQUERY_VIEWS_DATASET
+          value: "staging"
+        - name: INITIAL_S3_XML_FILE_LAST_MODIFIED_DATE
+          value: "2012-01-01 00:00:00"
+        - name: HEALTH_CHECK_URL
+          valueFrom:
+            secretKeyRef:
+              name: monitoring-urls
+              key: health_check_url
+        - name: DATA_HUB_MONITORING_SLACK_WEBHOOK_URL
+          valueFrom:
+            secretKeyRef:
+              name: monitoring-urls
+              key: slack_webhhok_url
+      sync:
+        resources:
+          requests:
+            memory: 400Mi
+            cpu: 800m
+
+    workers:
+      resources:
+        requests:
+          memory: "7Gi"
+          cpu: "800m"
+          ephemeral-storage: "10Gi"
+      extraVolumeMounts:
+      - name: data-hub-config-volume
+        mountPath: /dag_config_files/
+        readOnly: true
+      - name: gcloud-secret-volume
+        mountPath: /dag_secret_files/gcloud/
+        readOnly: true
+      - name: aws-secret-volume
+        mountPath: /home/airflow/.aws
+        readOnly: true
+      - name: gmail-secret-volume
+        mountPath: /dag_secret_files/gmail/
+        readOnly: true
+      - name: toggl-secret-volume
+        mountPath: /home/airflow/toggl
+        readOnly: true
+      - name: civi-secret-volume
+        mountPath: /home/airflow/civi_key
+        readOnly: true
+      - name: monitoring-urls-volume
+        mountPath: /dag_secret_files/monitoring_urls/
+        readOnly: true
+      - name: surveymonkey-secret-volume
+        mountPath: /dag_secret_files/surveymonkey/
+        readOnly: true
+
+      extraVolumes:
+      - name: data-hub-config-volume
+        configMap:
+          name: data-hub-configs
+      - name: gcloud-secret-volume
+        secret:
+          secretName: gcloud
+      - name: aws-secret-volume
+        secret:
+          secretName: credentials
+      - name: gmail-secret-volume
+        secret:
+          secretName: gmail-credentials
+      - name: toggl-secret-volume
+        secret:
+          secretName: toggl
+      - name: twitter-secret-volume
+        secret:
+          secretName: twitter
+      - name: civi-secret-volume
+        secret:
+          secretName: civi-key
+      - name: monitoring-urls-volume
+        secret:
+          secretName: monitoring-urls
+      - name: surveymonkey-secret-volume
+        secret:
+          secretName: surveymonkey-credentials
+    postgresql:
+      resources:
+        requests:
+          memory: 1Gi
+          cpu: 250m
+    flower:
+      resources:
+        requests:
+          memory: 150Mi
+          cpu: 50m
+    scheduler:
+      resources:
+        requests:
+          memory: 1500Mi
+          cpu: 800m
+    web:
```

Comment:
Another follow up might be to give this container a better name, I assume this is the 'web interface' for airflow.